### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by halloffame0793**
Writing the values to the console probably isn't needed but it fails with or without that.

### CODE REVIEW

File: Src/BootCamp.Chapter/Checks.cs [hunk 1]
```diff
@@ -16,26 +16,37 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            var Age = message;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs]: use consistent variable names. For local variables- camelCase (starts with non-capital)

File: Src/BootCamp.Chapter/Checks.cs [hunk 2]
```diff
@@ -16,26 +16,37 @@ public static class Checks
...
+            var ageConverted = int.Parse(Age);
+
+            Console.WriteLine(ageConverted);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs]: If it doesn't ask to print the age, don't print it.

File: Src/BootCamp.Chapter/Checks.cs [hunk 3]
```diff
@@ -16,26 +16,37 @@ public static class Checks
...
 
         public static float CalculateBmi(float weight, float height)
         {
-            // To do: call your implementation. 
-            return 0;
+            var inMeters = height * 100;
+            var metersSquared = Math.Pow(inMeters, 2);
+            var BMI = weight / metersSquared;
+
+            Console.WriteLine($"Your BMI is {BMI}");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs]: Calculating is not the same as printing. It should go to another line.

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -9,6 +9,25 @@ class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Please enter your name:");
+            var Name = Console.ReadLine();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Each of these 2 lines should go to the prompt methods.

Comment (Almantask) [reply Src/BootCamp.Chapter/Program.cs]: Because prompt means ask for. You ask for it in console, then read the user input form console.

Comment (halloffame0793) [reply Src/BootCamp.Chapter/Program.cs]: yup I refactored to include those lines in the methods. Doing it this way makes it so these prompt methods do not need to accept parameters, that may be a issue as the unit tests required these methods to accept params.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:13]: nitpick: no need an empty line right after a function's opening bracket.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+            Console.WriteLine("Please enter your height(cm):");
+            var height = Console.ReadLine();
+            var cHeight = Checks.PromptFloat(height);
+
+            Console.WriteLine("Please enter your weight(km):");
+            var weight = Console.ReadLine();
+            var cWeight = Checks.PromptFloat(weight);
+
+            Checks.CalculateBmi(cWeight, cHeight);
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:26]: nitpick: same here.

**Review by Almantask — APPROVED** [review]
Looks good :)

**Comment by Almantask** [comment]
Looks much better now!

**Pull request closed without merging**